backend/pkg/modules/internal/bfla/bfladetector/learn_and_detect_bfla.go

```diff
@@ -422,7 +451,7 @@ func (l *learnAndDetectBFLA) traceRunner(ctx context.Context, trace *CompositeTr
 func (l *learnAndDetectBFLA) notifyController(ctx context.Context, apiID uint) error {
```

Comment:
Are you sending a notification whenever new findings are added and upon reset?

Reply:
yes